src/app/pages/storage2/components/pools-dashboard/pools-dashboard.component.ts

```diff
@@ -86,8 +87,10 @@ export class PoolsDashboardComponent implements OnInit, AfterViewInit {
     this.store.loadDashboard();
 
     this.slideIn.onClose$
-      .pipe(untilDestroyed(this))
-      .subscribe(() => this.store.loadDashboard());
+      .pipe(
+        filter((value) => value.modalType === ImportPoolComponent && value.response === true),
```

Comment:
I don't think we need to do this. `onClose$` needs to be improved, but tying directly to `ImportPoolComponent` creates tightly coupled code here.